src/CGSOLAR.CPP

```diff
@@ -484,6 +484,7 @@ void ZNR::zn_SGClearTots()		// clear SG totals
 	}
 }		// ZNR::zn_SGClearTots
 //-----------------------------------------------------------------------------
+
```

Comment:
Not a big deal here, but whitespace changes should be intentional. I suspect these are incidental from reverting earlier changes. There are a lot of them in this PR and can distract from the actual changes that need reviewing.

Reply:
Ok, I'll go back and delete the white spaces.